modules/ob-defining-the-output.adoc

```diff
@@ -0,0 +1,94 @@
+// This module is included in the following assembly:
+//
+// shipwright-builds/configuring-openshift-builds.adoc
+
+:_content-type: CONCEPT
+[id="ob-defining-the-output_{context}"]
+= Defining the output
+
+In your `Build` CR, you can specify an output location to push the image. When using an external private registry as your output location, you must specify a secret to access the image. You can also specify the annotations and labels for the output image. 
+
+[NOTE]
+====
+When you specify annotations or labels, the output image will get pushed twice. The first push comes from the build strategy and the second push changes the image configuration to add the annotations and labels. 
+====
+
+The following example defines a public registry where the image will be pushed:
+
+[source,yaml]
+----
+apiVersion: shipwright.io/v1beta1
+kind: Build
+metadata:
+  name: s2i-nodejs-build
+spec:
+  source:
+    git:
+      url: https://github.com/shipwright-io/sample-nodejs
+      contextDir: source-build/
+  strategy:
+    name: source-to-image
+    kind: ClusterBuildStrategy
+  paramValues:
+  - name: builder-image
+    values: docker.io/centos/nodejs-10-centos7
+  output:
+    image: image-registry.openshift-image-registry.svc:5000/build-examples/nodejs-ex
+----
+
+The following example defines a private registry where the image will be pushed:
+
+[source,yaml]
+----
+apiVersion: shipwright.io/v1beta1
+kind: Build
+metadata:
+  name: s2i-nodejs-build
+spec:
+  source:
+    git:
+      url: https://github.com/shipwright-io/sample-nodejs
+      contextDir: source-build/
+  strategy:
+    name: source-to-image
+    kind: ClusterBuildStrategy
+  paramValues:
+  - name: builder-image
+    values: docker.io/centos/nodejs-10-centos7
+  output:
+    image: us.icr.io/source-to-image-build/nodejs-ex
+    credentials:
+      pushSecret: icr-knbuild
+----
+
+The following example defines annotations and labels for the image:
+
+[source,yaml]
+----
+apiVersion: shipwright.io/v1beta1
+kind: Build
+metadata:
+  name: s2i-nodejs-build
+spec:
+  source:
+    git:
+      url: https://github.com/shipwright-io/sample-nodejs
+      contextDir: source-build/
+  strategy:
+    name: source-to-image
+    kind: ClusterBuildStrategy
+  paramValues:
+  - name: builder-image
+    values: docker.io/centos/nodejs-10-centos7
+  output:
+    image: us.icr.io/source-to-image-build/nodejs-ex
+    credentials:
+      pushSecret: icr-knbuild
```

Comment:
```suggestion
    pushSecret: icr-knbuild
```
`pushSecret` replaces the `credentials.name` section completely. So, the format should be like:
~~~
...
output:
  image: <value>
  pushSecret: <value>
~~~